unstructured/metrics/table/table_eval.py

```diff
@@ -166,10 +194,8 @@ def process_file(self) -> TableEvaluation:
         )
 
         return TableEvaluation(
-            total_tables=total_tables,
-            table_level_acc=(
-                round(total_predicted_tables / total_tables, 2) if total_tables else np.nan
-            ),
+            total_tables=len(ground_truth_table_data),
+            table_level_acc=round(predicted_table_acc, 2),
             element_col_level_index_acc=metrics.get("col_index_acc", np.nan),
             element_row_level_index_acc=metrics.get("row_index_acc", np.nan),
             element_col_level_content_acc=metrics.get("col_content_acc", np.nan),
```

Comment:
BTW I was thinking also about content accuracy and whether we could use average levenshtein distance (this is how we actually measure CCT) instead of the current solution. Right now we're using `token_ratio`.

Reply:
definitely we should explore that as well; for this case I wanted to use similar algorithms we used to do matching a prediction to the actual table to compute the score so choose `difflib`. 
I don't know which will be better yet as the data has a lot of structural text in it (it is a dictionary) so token matching can be tricky (with structural infor tokens always matches).